tools/amd_build/patches/a_aten_src_ATen_Error.h.patch

```diff
@@ -0,0 +1,48 @@
+diff --git a/aten/src/ATen/Error.h b/aten/src/ATen/Error.h
+index b68779144..faea76bff 100644
+--- a/aten/src/ATen/Error.h
++++ b/aten/src/ATen/Error.h
+@@ -16,10 +16,10 @@
+ 
+ #include <stdarg.h>
+ 
+-#if !defined(_WIN32)
++#if !defined(__HIP_PLATFORM_HCC__)
```

Comment:
Looks like this will break the Windows build. Why can't you make it `!defined(_WIN32) && !defined(__HIP_PLATFORM_HCC__)`?